packages/@aws-cdk/aws-s3/test/integ.bucket.server-access-logs.ts

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+import cdk = require('@aws-cdk/core');
+import s3 = require('../lib');
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-s3-access-logs');
+
+const accessLogBucket = new s3.Bucket(stack, 'MyAccessLogsBucket', {
+  removalPolicy: cdk.RemovalPolicy.DESTROY,
+  accessControl: s3.BucketAccessControl.LOG_DELIVERY_WRITE,
```

Comment:
mmmm.... I'd expected this to be something that automatically happens when I use this bucket for server access logs.

Reply:
Yea, makes more sense.
Is this possible to achieve through the bucket's resource policy or it has to be through the Bucket's ACLs?
I looked at the docs and they mostly recommend setting them through the ACLs, I'll update the PR based on that but if you have a diff idea on how to enable the LogDelivery group, let me know :)